### PR TITLE
[CHK-2067] add api tests

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Add api tests to deploy

#### How Has This Been Tested?
https://dev.azure.com/pagopaspa/pagoPA-projects/_build/results?buildId=127574&view=results

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ resources:
     - repository: pagopaWalletTests
       type: github
       name: pagopa/pagopa-wallet-tests
-      ref: main
+      ref: CHK-2067-api-tests
```
Comment (giovanniberti) [inline .devops/deploy-pipelines.yml:93]: TODO revert this

**Review by giovanniberti — COMMENTED** [review]
No API tests before deploy in UAT? Or are they out of scope for this PR?

File: .devops/deploy-pipelines.yml (hunk 1 above)
Comment (ciuffagianluca) [reply .devops/deploy-pipelines.yml:93]: Yes, i will do after a test

**Comment by ciuffagianluca** [comment]
> No API tests before deploy in UAT? Or are they out of scope for this PR?

No need for test before deploy in UAT because in uat we will not deploy wallet-service by now

File: .devops/azure-templates/maven-github-release.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Node Github Relase steps
+# Mark a release on the project repository, with version bump and tag,
+# and publish a release on Github
+
+parameters:
```
Comment (pietro-tota) [inline .devops/azure-templates/maven-github-release.yml:5]: remove this file

**Pull request merged**